q2_phylogeny/plugin_setup.py

```diff
@@ -385,11 +385,15 @@
                              'present in at least 40% of the sequences.'
     },
     output_descriptions={
-        'aligned_seq': 'The aligned sequences.',
-        'masked_seq': 'The masked alignment.',
-        'unrooted_tree': 'The unrooted phylogenetic tree',
+        'alignment': 'The aligned sequences.',
+        'masked_alignment': 'The masked alignment.',
+        'tree': 'The unrooted phylogenetic tree',
```

Comment:
Missing a period at the end of this sentence. I don't care if they end in periods or not, but the other three outputs do.